JitsiParticipant.js

```diff
@@ -130,6 +157,19 @@ export default class JitsiParticipant {
         return this._role;
     }
 
+    /**
+     * Returns the sources for a given media type.
+     * @param {string} mediaType The type of media, 'audio' or 'video'.
+     * @returns {Map<string, Object>}
+     */
+    getSources(mediaType) {
+        if (!mediaType) {
```

Comment:
When would be need this? It feels weird to have 2 types of returns...

Reply:
JM calls both `getSources()` and `getSources('video')`.  I added the first case as an after thought. I will return the map here and the caller of the function can do further processing to get the video sources.